technische-universitat-dresden-betriebswirtschaftslehre-rechnungswesen-wirtschaftsprufung-und-steuerlehre.csl

```diff
@@ -0,0 +1,1511 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded" demote-non-dropping-particle="never" default-locale="de-DE">
+  <info>
+    <title>Technische Universität Dresden - Betriebswirtschaftslehre/Rechnungswesen/Wirtschaftsprüfung und Steuerlehre</title>
```

Comment:
The title should include ` (Deutsch)` at the end. 
See how our other styles have that: https://www.zotero.org/styles?q=Deutsch

Reply:
Thanks for your feedback, I adjusted this in my latest commit.